### PR TITLE
Metadata: ProtonTechnologies.ProtonVPN version 1.16.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21686)

### CODE REVIEW

File: manifests/p/ProtonTechnologies/ProtonVPN/1.16.3/ProtonTechnologies.ProtonVPN.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ShortDescription: High-speed Swiss VPN that safeguards your privacy.
+Description: We built ProtonVPN with extra security features to make it better at protecting your privacy than an ordinary VPN. When you use ProtonVPN to browse the web, your Internet connection is encrypted. By routing your connection through encrypted tunnels, ProtonVPNs advanced security features ensure that an attacker cannot eavesdrop on your connection. It also allows you to access websites that might be blocked in your country.
+Moniker: protonvpn
+Tags:
+- vpn
+- privacy
+- security
+- private
+- network
+- private network
```
Comment (ItzLevvie) [inline manifests/p/ProtonTechnologies/ProtonVPN/1.16.3/ProtonTechnologies.ProtonVPN.locale.en-US.yaml:26]:
```suggestion
- private-network
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please wait for an engineer to take a look.

**Comment by vedantmgoyal2009** [comment]
Ok

**Comment by vedantmgoyal2009** [comment]
Please look this PR also.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm seeing:
image

It seems to install the application using multiple MSI installers and one of them errors out because only one MSI installer can be installed at a time.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm still seeing the same on Windows Sandbox:
image

**Comment by vedantmgoyal2009** [comment]
If it is failing, the manifest should be removed from the repo.

**Pull request closed without merging**